packages/docusaurus-theme-translations/locales/uz/theme-common.json

```
@@ -0,0 +1,72 @@
+{
+  "theme.AnnouncementBar.closeButtonAriaLabel": "Yopish",
+  "theme.BackToTopButton.buttonAriaLabel": "Yuqoriga qaytish",
+  "theme.CodeBlock.copied": "Nushalandi",
+  "theme.CodeBlock.copy": "Nushalash",
+  "theme.CodeBlock.copyButtonAriaLabel": "Buferga nusxalash",
+  "theme.CodeBlock.wordWrapToggle": "Chiziqni oʻrashni oʻzgartirish",
+  "theme.DocSidebarItem.toggleCollapsedCategoryAriaLabel": "'{label}' turkumini yig‘ish/kengaytirish",
+  "theme.ErrorPageContent.title": "Sahifa buzildi.",
+  "theme.ErrorPageContent.tryAgain": "Yana bir bor urinib ko `ring",
+  "theme.NavBar.navAriaLabel": "Main",
+  "theme.NotFound.p1": "Afsuski, siz qidirayotkan sahifa topilmadi",
+  "theme.NotFound.p2": "Iltimos, ushbu havolaga ulangan sayt egasiga havola ishlamayotganligi haqida xabar bering.",
+  "theme.NotFound.title": "Sahifa topilmadi",
+  "theme.TOCCollapsible.toggleButtonLabel": "Ushbu sahifaning mazmuni",
+  "theme.admonition.caution": "ogohlantirish",
+  "theme.admonition.danger": "ehtiyot boling",
+  "theme.admonition.info": "ma'lumotingiz uchun",
+  "theme.admonition.note": "eslatma",
+  "theme.admonition.tip": "maslahat",
+  "theme.blog.archive.description": "Arxiv",
+  "theme.blog.archive.title": "Arxiv",
+  "theme.blog.paginator.navAriaLabel": "Bloglar roʻyxati sahifalarida navigatsiya",
+  "theme.blog.paginator.newerEntries": "Keyingi yozma",
+  "theme.blog.paginator.olderEntries": "Oldingi yozma",
+  "theme.blog.post.paginator.navAriaLabel": "Blog postlari sahifalarida navigatsiya",
+  "theme.blog.post.paginator.newerPost": "Keyingi post",
+  "theme.blog.post.paginator.olderPost": "Oldingi post",
+  "theme.blog.post.plurals": "{count} rekord|{count} yozuvlar|{count} yozuvlar",
+  "theme.blog.post.readMore": "Ko'proq o'qish",
+  "theme.blog.post.readMoreLabel": "{title} haqida batafsil",
+  "theme.blog.post.readingTime.plurals": "{readingTime} min. oqish",
+  "theme.blog.sidebar.navAriaLabel": "Oxirgi blog xabarlarini navigatsiya qilish",
+  "theme.blog.tagTitle": "“{tagName}” tegli {nPosts}",
+  "theme.colorToggle.ariaLabel": "Qorongʻi va yorugʻlik rejimini almashtirish (hozirda {mode} ishlatilmoqda)",
+  "theme.colorToggle.ariaLabel.mode.dark": "Tungi rejim",
+  "theme.colorToggle.ariaLabel.mode.light": "Kungi rejim",
+  "theme.common.editThisPage": "Sahifani tahrirlash",
+  "theme.common.headingLinkTitle": "{heading} ga bevosita havola",
+  "theme.common.skipToMainContent": "Asosiy tarkibga o'tish",
+  "theme.docs.DocCard.categoryDescription": "{count} element|{count} ta element|{count} ta element",
```

```suggestion
  "theme.docs.DocCard.categoryDescription": "{count} element|{count} ta element",
```

CF my previous review comment: you should only have 2 messages and not 3 here.

(and the other places as well)